spec/unit/views/components/paginated_collection_spec.rb

```diff
@@ -215,7 +215,7 @@ def paginated_collection(*args)
           expect(collection).not_to receive(:num_pages)
           expect(collection).not_to receive(:total_pages)
           pagination = paginated_collection(collection, pagination_total: false)
-          info = pagination.find_by_class('pagination_information').first.content.gsub('&nbsp;',' ')
+          info = pagination.find_by_class('pagination-information').first.content.gsub('&nbsp;',' ')
```

Comment:
Line is too long. [100/80]<br>Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.